exec/java-exec/src/test/java/org/apache/drill/TestPartitionFilter.java

```diff
@@ -362,4 +362,13 @@ public void testLogicalDirPruning2() throws Exception {
     testIncludeFilter(query, 1, "Filter", 10);
   }
 
-}
\ No newline at end of file
+  @Test  //DRILL-4665: Partition pruning should occur when LIKE predicate on non-partitioning column
+  public void testPartitionFilterWithLike() throws Exception {
```

Comment:
This test looks fine.  Could you also add one where LIKE predicate is on the partitioning column (it seems we don't have such a unit test).